tournaments/index.js

```diff
@@ -493,6 +497,28 @@ Tournament = (function () {
 		return true;
 	};
 
+	Tournament.prototype.setAutoStartTimeout = function (timeout, output) {
+		if (this.isTournamentStarted) {
+			output.sendReply('|tournament|error|AlreadyStarted');
+			return false;
+		}
+		if (timeout < AUTO_START_MINIMUM_TIMEOUT || isNaN(timeout)) {
+			output.sendReply('|tournament|error|InvalidAutoStartTimeout');
+			return false;
+		}
+
+		timeout = parseFloat(timeout);
```

Comment:
Oh, that's right, Javascript typecasting exists (even if it's disgusting)

It was more on the readability, so you show that it's going to be a `float` before you compare.

Heck, if it's set up to always make it either a number or `NaN` I wonder if you even need to call `parseFloat` at all